### PR TITLE
[dper3] Create dper LearningRate low-level module

### PR DESCRIPTION
**Pull request by esqu1**
Summary: As title; this will unblock migration of several modules that need learning rate functionality.

Test Plan:
```
buck test //dper3/dper3/modules/low_level_modules/tests:learning_rate_test
```

Differential Revision: D23681733

### CODE REVIEW

File: caffe2/sgd/learning_rate_op.cc [hunk 1]
```diff
@@ -163,8 +163,10 @@ C10_EXPORT_CAFFE2_OP_TO_C10_CPU(
     "int? inactive_period = -1, "
     "int? max_iter = -1, "
     "int? num_iter = 0, "
+    "float? decay = 1.0, "
     "float? start_multiplier = 0, "
     "float? end_mulitplier = 0, "
+    "float? end_multiplier = 0, "
```
Comment (houseroad) [inline caffe2/sgd/learning_rate_op.cc:169]: wait, why we double end_multiplier?

File: caffe2/sgd/learning_rate_op.cc [hunk 2]
```diff
@@ -163,8 +163,10 @@ C10_EXPORT_CAFFE2_OP_TO_C10_CPU(
     "int? inactive_period = -1, "
     "int? max_iter = -1, "
     "int? num_iter = 0, "
+    "float? decay = 1.0, "
```
Comment (houseroad) [inline caffe2/sgd/learning_rate_op.cc:166]: please add new parameter at the end, and make sure the default value matches the original behavior.

**Review by houseroad — COMMENTED** [review]
Current change is indeed BC breaking.

File: caffe2/sgd/learning_rate_op.cc (hunk 1 above)
Comment (esqu1) [reply caffe2/sgd/learning_rate_op.cc:169]: I had this because I thought that deleting the original typo'ed end_multiplier would break backwards compatibility; is it safe to just rename, or is there something else that has to be done?
